app/Http/Requests/Main/ContactUsRequest.php

```diff
@@ -24,9 +24,11 @@ public function authorize()
     public function rules()
     {
         return [
-            'txtname' => 'required|string|min:2|max:50',
-            'txtemail' => 'required|string|email|max:255',
-            'txtmessage' => ['required','max:512','min:5'],
+            'name' => ['required', 'string', 'min:2', 'max:50'],
+            'email' => ['required', 'string', 'email', 'max:255'],
+            'subject' => ['required', 'string', 'max:255'],
+            'message' => ['required', 'string', 'max:255'],
+            'mobile_number' => ['required','numeric'],
```

Comment:
Have you check this working or not?

Reply:
Also, I have a question why we are using numeric?

Reply:
Yes i have Checked it its working and to save just number in contact us form in front end  that's why I've just used numeric